sdf/1.11/inertial.sdf

```diff
@@ -5,10 +5,25 @@
     properties, and optionally its fluid added mass.
   </description>
 
+  <attribute name="auto" type="bool" default="false" required="0">
+    <description>
+      Set to true if you want automatic computation for the moments of inertia (ixx, iyy, izz) 
+      and products of inertia(ixy, iyz, ixz). Default value is false.
+    </description>
+  </attribute>
+
   <element name="mass" type="double" default="1.0" required="0">
     <description>The mass of the link.</description>
   </element>
 
+  <element name="density" type="double" default="1000.0" required="0">
+    <description>
+      Mass Density of the collision geometry. 
+      This is used to determine mass and inertia values during automatic calculation.
```

Comment:
Can you add that `density` values in `<collision>` elements will override this?

Reply:
Update the description in [ff88720](https://github.com/gazebosim/sdformat/pull/1299/commits/ff887207bf68326e12fa75e525e97eb8cbe7a3f8)